src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(new ColorSupplier().getRandomColor(), random.nextInt(100));
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You are creating a new instance of `ColorSupplier` every time `getRandomBall` is called. Instead, you should create a single instance of `ColorSupplier` and reuse it.